dashboard/config/linux/bits/net.yml

```diff
@@ -285,7 +285,7 @@ config:
  - VLAN_8021Q_MVRP
  - LLC2
  - NET_SCHED
- - NET_SCH_CBQ: [-v6.3]
```

Comment:
Removal of these was backported to stable, right?

Reply:
Yes